UpskillWikipediaUITests/PageObjects/SettingsPage.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+class SettingsPage: BaseAppPage {
+    
+    // MARK: - Elements
+    private lazy var privacyPolicyButton = app.staticTexts["Privacy policy"].firstMatch
+    private lazy var termsOfUseButton = app.staticTexts["Terms of Use"].firstMatch
+    
+    // MARK: - Actions
+    
+    func tapOpenPrivacyPolicy() {
+        _ = privacyPolicyButton.waitForExistence(timeout: TestTimeouts.generalUiElementTimeout)
+        privacyPolicyButton.tap()
```

Comment:
Thrid time already this code is repeated:
```
wait for element
tap an element
```
What about creating a general method or an `XCUIElement` extension function that waits for element and taps it afterwards?

Reply:
Corrected as suggested